packages/data-lake-api/scripts/checkTestDataLakeExists.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+DIR=$(pwd "$0")
+source $DIR/../../scripts/bash/mergeCurrentEnvWithEnvFile.sh 
+
+# Set default port in case it wasn't in .env
+: "${DATA_LAKE_DB_PORT:=5432}"
+
+if [ "$(PGPASSWORD=$DB_PG_PASSWORD psql -p $DATA_LAKE_DB_PORT -X -A -h $DATA_LAKE_DB_URL -U $DB_PG_USER -t -c "SELECT 1 FROM pg_database WHERE datname='$DATA_LAKE_DB_NAME'" )" = '1' ]
+then
+    exit 0
+fi
+
+echo -e "Error: $DATA_LAKE_DB_NAME database does not exist!\n\nTo create it, please run:\nyarn workspace @tupaia/data-lake-api setup-test-data-lake\n"
+exit 1
```

Comment:
```suggestion
echo -e "Error: $DATA_LAKE_DB_NAME database does not exist!\n\nTo create it, please get .env secret from lastpass then run:\nyarn workspace @tupaia/data-lake-api setup-test-data-lake\n"
exit 1
```

^ A minor suggestion, that helps my first time setup in local.

Reply:
Ah good call!